typescript/packages/subsurface-viewer/src/layers/axes2d/axes2DLayer.ts

```diff
@@ -53,8 +53,14 @@ const zDepthAxes = 0;
 const tickLineLength = 10;
 
 export interface Axes2DLayerProps extends ExtendedLayerProps {
-    marginH: number;
-    marginV: number;
+    /** Minimal horizontal pixel size margin. May be larger if this number is to small for the lable.
+     */
+    minimalMarginH: number;
+    /** Minimal vertical pixel size margin. May be larger if this number is to small for the lable.
+     */
+    minimalMarginV: number;
+    marginH: number; // @deprecated Use "mininalMarginH"
+    marginV: number; // @deprecated Use "mininalMarginV"
```

Comment:
I don't think the comment format will work. Check https://tsdoc.org/pages/spec/tag_kinds/